### ISSUE TITLE
Ports list

### ISSUE THREAD

**Issue by yoiang** [chancejs/chancejs#441]
**Chance** is such a wonderfully useful library, if there are any ports it'd be super useful to link them in README. And for languages/platforms where there isn't a port yet perhaps part of the doc should just mention if/how much they're appreciated!

...all this meaning it might be an effort myself and I'm sure others would be happy to contribute to ;)

**Comment by PhiLhoSoft** [comment]
What do you mean by "ports"?
Rewriting the library to another programming language?

**Comment by yoiang** [comment]
Yes, I believe that was my thinking! For those with the fortitude to do it if it was something y'all were open to having out there (maybe at the time I had a need for this functionality and breadth of types of random the library does but in another language I was considering it). 

Re effort of porting there are smarter ways to make them, tools that can create some automation between so updates are more easily made and less human-error prone, etc

**Comment by PhiLhoSoft** [comment]
One way to do it would be to start to extract the big data (lists of names, etc.) to Json format (for example), so it can be read by any language out there.
Converting the generators is relatively simple for most of them, then (but quite some work).

**Comment by yoiang** [comment]
🤘 awesome! So I think my original question was whether y'all were open to an effort like that if I should have a need in the future again that would give me the motivation to do that process
